src/components/Maps.vue

```diff
@@ -305,17 +308,20 @@ export default Vue.extend({
     },
     passModel(model) {
       this.selectedModel = model;
+    },
+    passMap(model) {
+      console.log(model);
```

Comment:
This argument should probably be `map`. The `console.log` is to be replaced later?

Reply:
Nah this was a left over statement which I've since removed.